packages/shared_preferences/lib/method_channel_shared_preferences.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:async';
+import 'package:flutter/services.dart';
+import 'shared_preferences_platform_interface.dart';
+
+const MethodChannel _kChannel =
+    MethodChannel('plugins.flutter.io/shared_preferences');
+
+/// Wraps NSUserDefaults (on iOS) and SharedPreferences (on Android), providing
+/// a persistent store for simple data.
+class MethodChannelSharedPreferences extends SharedPreferencesPlatform {
+  @override
+  Future<Map<String, Object>> getAll() =>
```

Comment:
Flutter Style is to only use `=>` if everything including the function declaration fits in one line: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods